src/WillowAuth.php

```diff
@@ -23,12 +25,11 @@ public static function instance()
 
     private function __construct()
     {
-        add_action('rest_api_init', [__CLASS__, 'registerApiControllers']);
-    }
+        $client = new Client([
+            'base_uri' => getenv('WILLOW_AUTH_ENDPOINT')
```

Comment:
I think it's better to use laravels `env()`, because it has a default param as the second param